server/utils/spellcheck_test.go

```diff
@@ -0,0 +1,63 @@
+package utils_test
+
+import (
+	"testing"
+
+	"github.com/runatlantis/atlantis/server/utils"
+	. "github.com/runatlantis/atlantis/testing"
+)
+
+func Test_Spellcheck(t *testing.T) {
+	t.Log("check if given executable name is misspelled or just an unrelated word")
+
+	spellings := []struct {
+		Misspelled bool
+		Given      string
+		Want       string
+	}{
+		{
+			false,
+			"atlantis",
+			"atlantis",
+		},
+		{
+			false,
+			"maybe",
+			"atlantis",
+		},
+		{
+			false,
+			"atlantis-qa",
+			"atlantis-prod",
+		},
+		{
+			true,
+			"altantis",
+			"atlantis",
+		},
+		{
+			true,
+			"atlants",
+			"atlantis",
+		},
+		{
+			true,
+			"teraform",
+			"terraform",
+		},
+	}
+
+	for _, s := range spellings {
+		isMisspelled := utils.Spellcheck(s.Given, s.Want)
+
+		if s.Misspelled {
+			Equals(t, isMisspelled, true)
+		}
+
+		if !s.Misspelled {
+			Equals(t, isMisspelled, false)
+		}
+
```

Comment:
Could you change this test using table driven tests using `t.Run`?

```go
  	for _, s := range spellings {
      t.Run(fmt.Sprintf("given %s want %s", s.Given, s.Want), func(t *testing.T) {
        isMisspelled := utils.Spellcheck(s.Given, s.Want)
        ...
      })
    }
```

Reply:
done